_deploy/r/gnoswap/gnft/gnft.gno

```diff
@@ -84,61 +135,59 @@ func Approve(user pusers.AddressOrName, tid grc721.TokenID) {
 	}
 }
 
+// SetApprovalForAll enables or disables approval for a third party to manage all tokens
+// Parameters:
+//   - user: The address or username to set approval for
+//   - approved: true to approve, false to revoke approval
 func SetApprovalForAll(user pusers.AddressOrName, approved bool) {
 	common.IsHalted()
-
-	err := gnft.SetApprovalForAll(users.Resolve(user), approved)
-	if err != nil {
-		panic(err.Error())
-	}
+	checkErr(gnft.SetApprovalForAll(users.Resolve(user), approved))
 }
 
+// TransferFrom transfers ownership of a token from one address to another
+// Parameters:
+//   - from: The current owner's address or username
+//   - to: The recipient's address or username
+//   - tid: The token ID to transfer
 func TransferFrom(from, to pusers.AddressOrName, tid grc721.TokenID) {
```

Comment:
After the owner is changed with transferfrom for tokenId, the tokenList should also be further updated (deleted from the address in from, added to the address in to).

Reply:
53464b183d9fcfbf86fc280162eb5b8f86b66da6